sdk/avalon_sdk/work_order/work_order_params.py

```diff
@@ -99,20 +97,28 @@ def set_workload_id(self, workload_id):
 
     def set_requester_id(self, requester_id):
         """Set requesterId work order parameter."""
+        if requester_id is None or \
+                not is_valid_hex_str(requester_id) or \
+                len(requester_id) != 64:
```

Comment:
Instead of hard coded value, use constant in the class.

Reply:
Added length check for params using constant in proxy model.